### PR TITLE
Add "memuplay.com"

### PR DESCRIPTION
**Pull request by thecodymetalist**
memuplay_before

memuplay_after

Tested on this subdomain : [https://www.memuplay.com/blog/en](https://www.memuplay.com/blog/en)

### CODE REVIEW

**Comment by Gusted** [comment]
It's not dark-by-default.

image

**Comment by Gusted** [comment]
I'm sorry for that, I'd messed up while reviewing another PR.

**Review by Gusted — APPROVED** [review]

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -10816,6 +10816,26 @@ body {
 
 ================================
 
+magnetdl.com
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10819]: Please re-order this fix in alphabetic order.

**Comment by thecodymetalist** [comment]
> I'm sorry for that, I'd messed up while reviewing another PR.

No problem 👍🏻

**Pull request closed without merging**